docs/usage/quickstart.rst

```diff
@@ -247,3 +258,5 @@ services and their configuration.
 .. _Grafana: https://grafana.com
 .. _Cerebro: https://github.com/lmenezes/cerebro
 .. _Kibana: https://www.elastic.co/products/kibana/
+.. _storage: (Link needed)
+.. _releases: https://github.com/scality/metalk8s/releases
```

Comment:
I tend to put references closed to where they're used rather than at the bottom of the file, far from their use site. This eases reading/reviewing the ReST source.